pkg/model/awsmodel/firewall.go

```diff
@@ -193,21 +196,33 @@ func (b *FirewallModelBuilder) applyNodeToMasterBlockSpecificPorts(c *fi.Cloudup
 			for _, protocol := range protocols {
 				awsName := strconv.Itoa(int(protocol))
 				name := awsName
+
 				switch protocol {
 				case ProtocolIPIP:
 					name = "ipip"
+					t := &awstasks.SecurityGroupRule{
+						Name:          fi.PtrTo(fmt.Sprintf("node-to-master-protocol-%s%s", name, suffix)),
+						Lifecycle:     b.Lifecycle,
+						SecurityGroup: masterGroup.Task,
+						SourceGroup:   nodeGroup.Task,
+						Protocol:      fi.PtrTo(awsName),
+					}
+					AddDirectionalGroupRule(c, t)
+				case ProtocolICMP:
+					name = "icmp"
+					t := &awstasks.SecurityGroupRule{
+						Name:          fi.PtrTo(fmt.Sprintf("node-to-master-protocol-%s%s", name, suffix)),
+						Lifecycle:     b.Lifecycle,
+						SecurityGroup: masterGroup.Task,
+						SourceGroup:   nodeGroup.Task,
+						FromPort:      fi.PtrTo(int64(8)),
```

Comment:
nit: I think `net` may have some enums that can be used here instead of magic number 8

Reply:
You're right makes more sense to use the embedded golang ICMPType constants rather than the raw number.